environment-setup.md

```diff
@@ -117,10 +117,13 @@ If you need to work with multiple versions of node, you may skip installing node
 To install a **JDK** run the following command:
 
 ```cli
-# todo: check if it works with latest jdk
 brew install --cask adoptopenjdk
```

Comment:
I guess since `brew install --cask adoptopenjdk` now installs JDK 16 - which isn't yet supported by gradle, we should by default recommend installing a supported version:

```suggestion
brew tap AdoptOpenJDK/openjdk
brew install --cask adoptopenjdk14
```

I'd need to check if `adoptopenjdk15` is supported - and if it is, use that instead.

Reply:
Yes, you are right.

My mistake is to only run `brew search adoptopenjdk`, where two packages are found:

- adoptopenjdk (latest)
- adoptopenjdk8

Add third-party repository with `brew tap` was unknown for me.

I will test in 8 hours with adoptopenjdk14.

Reply:
@bgrand-ch thank you! I couldn't find `adoptopenjdk8` on https://brew.sh/ search, that lead me to find the third-party (they provide the official `adoptopenjdk` too, just not "polluting" the registry/list with all versions unless you tap into theirs).

If you could test `adoptopenjdk15` first, that would be ideal to suggest using the most recent supported versions.

Reply:
@rigor789 some test results, in order:

1) I have [uninstalled](https://stackoverflow.com/a/66719581/10531083) all **adoptopenjdk** on my Mac, check with `brew list` and restart the Mac. When I launch my "Pixel 4 XL API 30" AVD with Android Studio 4.1.3 and run `ns run android` on my project, all works fine, without any **adoptopenjdk** Homebrew formulae installed.

2) I have [installed](https://github.com/AdoptOpenJDK/homebrew-openjdk) **adoptopenjdk15** on my Mac, check with `brew list` and restart the Mac. When I launch my AVD with Android Studio and run `ns run android` on my project, all works fine with **adoptopenjdk15**.

3) I have [uninstalled](https://stackoverflow.com/a/66719581/10531083) all **adoptopenjdk** on my Mac, check with `brew list` and restart the Mac. I have [installed](https://github.com/AdoptOpenJDK/homebrew-openjdk) **adoptopenjdk** ([latest](https://formulae.brew.sh/cask/adoptopenjdk)) on my Mac, check with `brew list` and restart the Mac. When I launch my AVD with Android Studio and run `ns run android` on my project, all works fine with **adoptopenjdk** (latest).

---

**And I realize that I didn't run the `ns clean` command between all tests above...**

If I run `ns clean` between all tests above: 

1) There is the error "Unable to apply changes on device: emulator-5554. Error is: Command ./gradlew failed with exit code 1.".

2) Works like a charm 👍 

3) There is the error:
```
Execution failed for task ':app:processDebugManifest'.
Unable to make field private final java.lang.String java.io.File.path accessible: module java.base does not "opens java.io" to unnamed module @4b133842

Unable to apply changes on device: emulator-5554. Error is: Command ./gradlew failed with exit code 1.
```